testcase/resolve-lock/resolve_lock.go

```diff
@@ -540,7 +552,9 @@ func (c *resolveLockClient) reset(ctx context.Context) {
 
 func (c *resolveLockClient) getTs(ctx context.Context) (uint64, error) {
 	physical, logical, err := c.pd.GetTS(ctx)
-	if err != nil {
+	//if pd down, errmsg is io.EOF,testcase ignore this errmsg
+	//panic_check plugin check pd panic
+	if err != nil && err != io.EOF {
```

Comment:
It's not suitable to keep running the test if it happens.

Reply:
If the error is `EOF`, it means PD is down. I think we needn't handle such a case. It must be a bug.